### DISCUSSION TITLE
A big thank you for Courseplay v6.4.0.3

### DISCUSSION THREAD

**Discussion by harleymac** [Courseplay/courseplay#7318] [General]
Thanks to all who worked in their private time on this new release.
This is really a big step forward. I have never seen such a fast unloading of the combine harvesters. 
The tractors with trailers are starting right in the second, when the combine harvester is in unloading postition and they have a much faster navigation to the target now.
Great achievement!

Kind regards

**Comment by Tensuko** [comment]
Well... nice to hear that :)

**Comment by JustDen1234** [comment]
I wanted to comment also about new version of CP. Now using 6.4.0.5 and do not have to change anything myself after generating a course to harvest a field. And it does not miss any fruit, fields are clean after harvesting. 

The way the turns work now is just incredible !!!!!
